iaso/api/setup_account.py

```diff
@@ -52,6 +51,21 @@ def create(self, validated_data):
             user_manual_path=validated_data.get("user_manual_path"),
         )
 
+        # Create a setup_account project with an app_id represented by the account name
+        initial_project = Project.objects.create(
+            name=validated_data["account_name"] + " project", account=account, app_id=validated_data["account_name"]
+        )
+
+        # Create an initial orgUnit type and link it to project
+        initial_orgunit_type = OrgUnitType.objects.create(name="COUNTRY", short_name="country", depth=1)
+        initial_orgunit_type.projects.set([initial_project])
```

Comment:
I'm not sure about this. Will all projects have "COUNTRY" as org unit type? If not, will it not mess up when importing a data source?

Reply:
Also, does it have to be shouting at the user? :sweat_smile: 

I think the name would be much better as "Country"

Reply:
> 

The idea of the ticket is to create a complete account setup so that the user can directly create an orgUnit, the Country is the default orgUnit type.